src/components/scripture/scripture-range.resolver.ts

```diff
@@ -26,8 +27,8 @@ export class ScriptureRangeResolver {
   @ResolveField(() => Int, {
     description: 'The total number of verses in this scripture range',
   })
-  totalVerses(@Parent() _range: ScriptureRange): number {
-    // TODO
-    return 0;
+  totalVerses(@Parent() range: ScriptureRange): number {
+    const verseRange = scriptureToVerseRange(range);
+    return Math.abs(verseRange.end - verseRange.start) + 1;
```

Comment:
We shouldn't need to abs. End should always come after start. I'll create an issue to enforce this.

Reply:
Sure, got it